src/ZendeskApi.Client/Resources/Interfaces/ITagsResource.cs

```diff
@@ -0,0 +1,21 @@
+using System.Threading;
+using System.Threading.Tasks;
+using ZendeskApi.Client.Models;
+using ZendeskApi.Client.Responses;
+
+namespace ZendeskApi.Client.Resources.Interfaces
+{
+    public interface ITagsResource
+    {
+        #region List
+        /// <summary>
+        /// Lists all tags. This request is paginated.
+        /// </summary>
+        Task<TagResponse> GetAllAsync(
+            CursorPager cursor = null,
```

Comment:
i don't think we want to give consumers the option to not pass it in. I don't think a cursor request makes sense without this object supplied